bindsnet/models/__init__.py

```diff
@@ -110,29 +111,26 @@ def __init__(self, n_inpt: int, n_neurons: int = 100, exc: float = 22.5, inh: fl
 class CANs(Network):
     # language=rst
     """
-    Creates a number of capillary-astrocyte neuron (CAN) units.
+    Defines a number of capillary-astrocyte neuron (CAN) units.
     """
 
-    def __init__(self, n_inpt: int, n_neurons: int = 1000, n_CANs: int = 4, dt: float = 1.0,
-                 running_time: int = 500) -> None:
+    def __init__(self, n_inpt: int, n_neurons: int = 1000, n_cans: int = 4, dt: float = 1.0) -> None:
         # language=rst
         """
         :param n_inpt: Number of input neurons. Matches the 1D size of the input data.
         :param n_neurons: Number of neurons in each CAN unit.
         :param n_CANs: Number of CAN units.
         :param dt: Simulation time step.
-        :param running_time: Minimum iteration to run for memory allocations.
         """
         super().__init__(dt=dt)
 
         self.n_inpt = n_inpt
         self.n_neurons = n_neurons
-        self.n_CANs = n_CANs
-        self.running_time = running_time
+        self.n_cans = n_cans
 
         self.add_layer(Input(n=self.n_inpt), name='X')
```

Comment:
`self.add_layer(Input(n=self.n_inpt), name='X')`
should be `RealInput` insted of `Input` 
But its not running... discussion is needed